src/libraries/Common/src/System/SR.cs

```diff
@@ -23,11 +23,11 @@ internal static partial class SR
             false;
 #endif
 
-        internal static string GetResourceString(string resourceKey, string? defaultString = null)
+        internal static string GetResourceString(string resourceKey)
```

Comment:
This creates a difference from its .vb counterpart.

Reply:
Does it matter? The only place where it's used is in archived Microsoft.VisualBasic.Core library

Reply:
I don't know that it matters, but I wanted to call it out.
cc: @tarekgh

Reply:
> The only place where it's used is in archived Microsoft.VisualBasic.Core library

that is right. if this change is not affecting `Microsoft.VisualBasic.Core`, it should be ok. But should we just keep C# and VB match for consistency?

Reply:
I'd rather not touch Microsoft.VisualBasic.Core code and I don't think the consistency matters here.